src/es/components/molecules/systemNotification/SystemNotification.js

```diff
@@ -0,0 +1,159 @@
+// @ts-check
+import SystemNotification from '../../web-components-toolbox/src/es/components/molecules/systemNotification/SystemNotification.js'
+
+export default class KsSystemNotification extends SystemNotification {
+  constructor(options = {}, ...args) {
+    super({ ...options }, ...args)
+  }
+
+  connectedCallback() {
+    if (this.shouldRenderCSS()) this.renderCSS()
+    this.renderHTML()
```

Comment:
The shouldRenderHTML is missing here. Just check for a query selector eg. !this.root.querySelector('.system-notification') analog other components. Without this check, your component would render on every connect to the DOM. Some components need that behavior, but then ether clear their html each time or update accordingly.